### PR TITLE
csss fix mobile-desktop style

### PR DESCRIPTION
**Pull request by SemyonTrushik**

### CODE REVIEW

File: src/components/CreatePosts.vue [hunk 1]
```diff
@@ -43,12 +43,10 @@ export default {
   methods: {
     newPost() {
       const { title, content } = this;
-      createPost(title, content)
-        .catch(error => {
-          handleRequestError.call(this, error);
-        });
+      createPost(title, content).catch(error => {
+        handleRequestError.call(this, error);
```
Comment (dromix) [inline src/components/CreatePosts.vue:47]: I don't like a way you are calling an error

File: src/components/PostsList.vue [hunk 2]
```diff
@@ -52,20 +52,21 @@ export default {
     goEditPost(postId) {
       try {
         this.$router.push({ path: `/editPosts/${postId}` });
-      } catch (error) { /* empty */ }
+      } catch (error) {
+        /* empty */
+      }
```
Comment (dromix) [inline src/components/PostsList.vue:57]: remove try catch or add some logic here

File: src/components/TheBurger.vue [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<template>
+  <div
+    class="burger"
+    @click=" $emit('toggleSwitch')"
+    @keydown="bar"
+  >
+    <span>{{}}</span>
+    <span>{{}}</span>
+    <span>{{}}</span>
```
Comment (dromix) [inline src/components/TheBurger.vue:9]: What is it empty?

File: src/components/TheBurger.vue [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+<script>
+export default {
+  data() {
+    return {
+      switchToggle: true
+    };
+  },
+  methods: {
+    toggleSwitch() {
+      // eslint-disable-next-line no-console
```
Comment (dromix) [inline src/components/TheBurger.vue:22]: Remove comment

File: src/components/TheBurger.vue [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    width: 2rem;
+    height: 2rem;
+    @include for-desktop{
+      display: none;
+    }
+  }
+
+.burger span {
+    width: 100%;
+    border:  1px solid #fff;
```
Comment (dromix) [inline src/components/TheBurger.vue:45]: use rem

File: src/components/TheHeader.vue [hunk 6]
```diff
@@ -1,3 +1,4 @@
+<!-- eslint-disable vuejs-accessibility/click-events-have-key-events -->
```
Comment (dromix) [inline src/components/TheHeader.vue:1]: Remove all eslint disabled rules

File: src/components/TheRegistration.vue [hunk 7]
```diff
@@ -55,5 +55,4 @@ export default {
 };
 </script>
 
-<style>
-</style>
+<style></style>
```
Comment (dromix) [inline src/components/TheRegistration.vue:58]: remove

Comment (SemyonTrushik) [reply src/components/TheRegistration.vue:58]: ?

Comment (SemyonTrushik) [reply src/components/TheRegistration.vue:58]: что конкретно убрать то ?

File: .eslintrc.js [hunk 8]
```diff
@@ -135,7 +135,6 @@ module.exports = {
     ],
     'no-bitwise': ['error', { allow: ['~'] }],
     'no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
-    'max-len': ['error', { code: 120 }],
```
Comment (dromix) [inline .eslintrc.js:138]: why did you remove?

File: src/components/TheHeader.vue [hunk 9]
```diff
@@ -26,53 +27,126 @@
...
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 2rem 3rem;
+  a {
+    color: var(--all-text-white);
+    text-decoration: none;
+    font-size:3rem;
```
Comment (dromix) [inline src/components/TheHeader.vue:68]: Maybe need to use prettier also in format script or check if there some rules in eslint ( but I'm not sure if it exists for CSS) 
```suggestion
    font-size: 3rem;
```

File: src/components/TheRegistration.vue (hunk 7 above)
Comment (dromix) [reply src/components/TheRegistration.vue:58]: style block is empty why do you need this?

File: src/style/main.scss [hunk 10]
```diff
@@ -9,48 +9,6 @@
...
-// input {
-//   padding: 0.5rem;
-//   width: 15rem;
-//   height: 2rem;
-//   display: block;
-// }
\ No newline at end of file
+html {
+  background: var(--background-main-color);
+}
```
Comment (dromix) [inline src/style/main.scss:14]: Move it to base.scss

File: src/pages/TheHome.vue [hunk 11]
```diff
@@ -22,39 +25,60 @@ export default {
...
-      color: #A9ADC1;
+      display: block;
+      margin: 1rem auto 0;
+      width: 80%;
+      font-size: 2.5rem;
+      line-height: 3rem;
+      color: var(--subtitle-color);
+       @include for-desktop {
+        width: 100%;
+      }
```
Comment (dromix) [inline src/pages/TheHome.vue:76]: I would move it to typography.scss

File: .eslintrc.js (hunk 8 above)
Comment (SemyonTrushik) [reply .eslintrc.js:138]: ошибка длинны строки

File: src/components/TheHeader.vue [hunk 12]
```diff
@@ -1,78 +1,197 @@
-<!-- eslint-disable vuejs-accessibility/anchor-has-content -->
-<!-- eslint-disable max-len -->
+<!-- eslint-disable vuejs-accessibility/click-events-have-key-events -->
```
Comment (dromix) [inline src/components/TheHeader.vue:1]: mark todo to remove it later

File: src/components/TheHeader.vue [hunk 13]
```diff
@@ -1,78 +1,197 @@
...
+<!-- eslint-disable vue/html-button-has-type -->
 <template>
   <navbar class="header">
     <div class="name">
       <a href="">{{ userName }}</a>
     </div>
-    <div class="links">
+    <div
+      class="links"
+      :class="{ show: show }"
```
Comment (dromix) [inline src/components/TheHeader.vue:10]:
```suggestion
      :class="{ show }"
```

File: src/components/TheHeader.vue [hunk 14]
```diff
@@ -1,78 +1,197 @@
...
+    >
+      <button
+        class="close"
+        @click="showLinks"
+      >
+        Close
+      </button>
       <a href="#">Blog</a>
```
Comment (dromix) [inline src/components/TheHeader.vue:18]: use <router-link>

File: src/components/TheHeader.vue [hunk 15]
```diff
@@ -1,78 +1,197 @@
...
+    padding: 1rem 2rem;
+    justify-content: space-between;
+  }
+}
+.burger {
+  align-self: center;
+  @include for-desktop {
+    display: none;
+  }
+  img {
```
Comment (dromix) [inline src/components/TheHeader.vue:93]: It's better to use classes instead of native elements

File: src/components/TheHeader.vue [hunk 16]
```diff
@@ -1,78 +1,197 @@
...
+  }
+}
+.links a:hover {
+  text-decoration: underline;
+  color: var(--link-color);
+}
+.show {
+  display: flex;
+}
+// .links {
```
Comment (dromix) [inline src/components/TheHeader.vue:156]: remove dead code

**Review by dromix — APPROVED** [review]

File: src/components/TheHeader.vue (hunk 15 above)
Comment (SemyonTrushik) [reply src/components/TheHeader.vue:93]: +

File: src/components/TheHeader.vue (hunk 13 above)
Comment (SemyonTrushik) [reply src/components/TheHeader.vue:10]: +
